microprofile/server/src/main/java/io/helidon/microprofile/server/Main.java

```diff
@@ -32,12 +45,62 @@ private Main() {
      *
      * @param args command line arguments, currently ignored
      */
+    @SuppressWarnings("CallToPrintStackTrace")
     public static void main(String[] args) {
+        try {
+            configureLogging();
+        } catch (IOException e) {
+            System.err.println("Failed to configure logging");
+            e.printStackTrace();
+        }
+
         Server server = Server.create();
         server.start();
         port = server.port();
     }
 
+    private static void configureLogging() throws IOException {
+        String configPath = System.getProperty("java.util.logging.config.file");
+        String source = "defaults";
+
+        if (configPath == null) {
+            // first set up logging - but only if not configured on command line
```

Comment:
Suppose I use the [`java.util.logging.class` System property](https://docs.oracle.com/en/java/javase/12/docs/api/java.logging/java/util/logging/LogManager.html).  I think you need to account for that as well.  That might be a better way to implement this whole thing, actually.

Reply:
I now take the `.class` property into account. I do not see the advantage in creating a new class when we can handle this in the same method...